### DISCUSSION TITLE
Integrating Additional Networks

### DISCUSSION THREAD

**Discussion by BunsDev** [ApeWorX/ape#721] [Ideas]
Would it be permissible to construct a PR to integrate new networks, for example BSC, Fantom Opera, Avalanche, etc? I'd like to know before spending time on something that is forbidden. If this is not allowed / encouraged, I am curious whether this is a technical constraint or a value-based objection.

**Comment by fubuloubu** [comment]
Hi @BunsDev! We actually already have support for the BSC and Fantom ecosystems through our plugin system:
https://github.com/ApeWorX/ape-bsc
https://github.com/ApeWorX/ape-fantom

This is the suggested method for adding new ecosystems to ape!

**Reply by BunsDev** [reply]
Fantastic, thanks for sharing!

**Comment by fubuloubu** [comment]
Ape now has [Custom Networks](https://docs.apeworx.io/ape/stable/userguides/networks.html#custom-network-connection)
